pkg/ams/terms.go

```diff
@@ -0,0 +1,43 @@
+package ams
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"net/http"
+
+	"github.com/redhat-developer/app-services-cli/internal/build"
+)
+
+var fallbackTocSpec = []byte(`{
+    "kafka":{
+       "EventCode":"register",
+       "SiteCode":"ocm",
+       "StopOnTermsChange": true
+    },
+    "service-registry":{
+       "EventCode":"onlineService",
+       "SiteCode":"ocm",
+       "StopOnTermsChange": true
+    }
+  }
+`)
+
+func GetRemoteTermsSpec() TermsAndConditionsSpec {
+	response, err := http.Get(build.TermsReviewSpecURL)
```

Comment:
My understanding is that considering how frequent you would create instances using cache might result in potential. 

We do not cache AMS terms check so caching this would be overkill. 
I did added cache initially in login but it would be too complex and hard to verify/handle all the cases.

> Or perhaps have this as a JSON file which is embedded into the binary

I had that json as fallback but moved to structure and provided documentation on where is the source for the cdn file.